manila/templates/shares/_netapp-configmap.yaml.tpl

```diff
@@ -0,0 +1,17 @@
+{{- define "share_netapp_configmap" -}}
+{{- $context := index . 0 -}}
```

Comment:
It seems we are not using the global $context in this template so we could avoid the tuple foo in this case, no?